### ISSUE TITLE
garden cli fails when using -log global option but not when using --loglevel

### ISSUE THREAD

**Issue by Deepakkothandan** [garden-io/garden#151]
When running garden with the `-log` option, garden fails with the below error, but it works when using `--loglevel`

```
node ~/devel/node/garden/dist/src/bin/garden.js status -log verbose

USAGE

  garden status [options]

Global options
  -h, --help        Show help
                    [boolean]
....

  -log, --loglevel  Set logger level.
                    [enum] [default: info] [error, warn, info, verbose, debug, silly]


  -o, --output      Output command result in specified format (note: disables progress
                    logging).
                    [enum] [json, yaml]

Value "" is invalid for argument o or output. Choices are: json, yaml
```

**Comment by eysi09** [comment]
Hi 👋 @Deepakkothandan 

The correct syntax for non-boolean options is `garden <command> --option=value` (note the `=`sign). E.g.:

```
garden status -log=verbose
```
or
```
garden status --loglevel=verbose
```
If the option is boolean you can skip the value, e.g.:
```
garden build --force
``` 
However, that doesn't explain the inconsistent error message. So, thanks for pointing that out. I'll look into it.

P.S. Positional arguments don't use the `=` sign. So for completeness, the entire syntax looks like this: `garden <command> arg1 arg2 --opt1=val1 --opt2=val2l`. E.g.:

```
garden deploy hello-container --force --loglevel=verbose
```
The options can be placed anywhere, e.g. before the command or between the command and it's arguments.

**Comment by Deepakkothandan** [comment]
👋 @eysi09 Thank you for the clarification. But if this is the case then `--loglevel verbose` should also not work, right ? But running `garden status --loglevel verbose` works

**Comment by eysi09** [comment]
Yes, you're right. That's what I'm looking into.

**Comment by eysi09** [comment]
Ah, I see. It does appear that you can indeed skip the `=` sign when passing arguments to options. However, the parser only allows single character aliases for some reason. So the alias for `--loglevel` should be `-l`.

I think it makes sense to change the alias for `loglevel` to `l` for now and then have a discussion about whether we'll allow multi character aliases in the future. A single character seems to be the convention.

**Comment by Deepakkothandan** [comment]
@eysi09 yeah I can confirm that. I did that change already in the code, using single letter aliases and it does work without the `=` sign.

**Comment by eysi09** [comment]
PR with fix is up at #152
